src/EditorFeatures/Text/Extensions.SnapshotSourceText.cs

```diff
@@ -267,9 +268,9 @@ public override SourceText WithChanges(IEnumerable<TextChange> changes)
             private class ChangedSourceText : SnapshotSourceText
             {
                 private readonly SnapshotSourceText _baseText;
-                private readonly ITextSnapshot _baseSnapshot;
+                private readonly ITextImage _baseSnapshot;
```

Comment:
@CyrusNajmabadi Maybe that difference doesn't matter anymore now that we're not holding onto snapshots? If that's the case then maybe just clean it up if it's easy? Saves us a field at least on a fairly common type.